stablehlo/dialect/StablehloOps.td

```diff
@@ -2048,9 +2048,24 @@ def StableHLO_DotGeneralOp: StableHLO_ShapedInterfaceOp<"dot_general",
   let summary = "General Dot operator";
   let description = [{
     Performs general dot products between vectors, vector/matrix and
-    matrix/matrix multiplication.
+    matrix/matrix multiplication over dimensions specified in
+    `dot_dimension_numbers`.
 
-    See https://www.tensorflow.org/xla/operation_semantics#dotgeneral.
+    See:
+    https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md#stablehlodot_general
+
+    Example:
+    ```mlir
+    %0 = "stablehlo.dot_general"(%arg0, %arg1) {
```

Comment:
Let's use `%result`, `%lhs` and `%rhs` to be consistent with the Examples section above?

Reply:
Sure. I had them named as such since the other ops in the file is following this format. I'll make another PR renaming the examples to match the spec.